README.md

```diff
@@ -58,6 +58,7 @@ Polyfills are provided for:
 - the `Stringable` interface introduced in PHP 8.0;
 - the `PhpToken` class introduced in PHP 8.0 when the tokenizer extension is enabled;
```

Comment:
can you also add this line to the readme of Php80 please?

Reply:
`Stringable` interface is already  present in Php80 Readme ;-)  (https://github.com/symfony/polyfill/blame/main/src/Php80/README.md#L6) , but I'll add a link to PHP documentation (https://www.php.net/manual/en/class.stringable)

Reply:
I mean the line about PhpToken

Reply:
FYI: I've added more links to PHP Documentation with  commit https://github.com/symfony/polyfill/pull/401/commits/58b6df77562437e4772ba8524db6819bb756df2a